grails-app/controllers/org/pih/warehouse/SentryInterceptor.groovy

```diff
@@ -9,42 +9,60 @@
  **/
 package org.pih.warehouse
 
-
 import grails.util.Environment
-import io.sentry.SentryClient
-import io.sentry.event.UserBuilder
+import groovy.transform.CompileStatic
+import io.sentry.Sentry
+import io.sentry.protocol.User as SentryUser
 import org.pih.warehouse.core.Location
 import org.pih.warehouse.core.User
 
+@CompileStatic
 class SentryInterceptor {
-    SentryClient sentryClient
 
-    public SentryInterceptor() {
-        matchAll().except(uri: '/static/**').except(uri: "/info").except(uri: "/health")
+    SentryInterceptor() {
+        matchAll().except(uri: '/static/**').except(uri: '/info').except(uri: '/health')
```

Comment:
Can you turn this formatter off? These are the "minor" changes I want to avoid re: rebase conflicts. This isn't a problem here because interceptors are new to Grails 3, but I just keep seeing changes like this all over and knowing how painful rebasing is, this is just adding to our death by a thousand cuts rebasing problem.

Reply:
yeah, for sure, I'll clean this up and squash. I try to prevent "editor tidying" from creeping in and must have missed this one

Reply:
Fixed in 23788a183.